src/docs/cookbook-payments/removing-stripe/index.md

```diff
@@ -0,0 +1,332 @@
+when you are building the transaction process to your marketplace it might make
+sense to rename some of the states and transitions (e.g. preauthorized) so that
+they make more sense in your use-case.
+
+#### Edit util/transaction.js to reflect the changes in your transaction process
+
+In
+[src/util/transaction.js](https://github.com/sharetribe/flex-template-web/blob/master/src/util/transaction.js)
+file, there are lots of helper functions that are used to determine which state
+the transaction is. This file should be updated to match the new transaction
+process.
+[Read more about editing transaction.js](/cookbook-transaction-process/change-transaction-process-in-ftw/#2-check-if-the-transactionjs-file-needs-to-be-updated)
+
+Example:
+
+```diff
+     [STATE_INITIAL]: {
+       on: {
+         [TRANSITION_ENQUIRE]: STATE_ENQUIRY,
+-        [TRANSITION_REQUEST_PAYMENT]: STATE_PENDING_PAYMENT,
++        [TRANSITION_REQUEST_PAYMENT]: STATE_PREAUTHORIZED,
+       },
+     },
+     [STATE_ENQUIRY]: {
+       on: {
+-        [TRANSITION_REQUEST_PAYMENT_AFTER_ENQUIRY]: STATE_PENDING_PAYMENT,
++        [TRANSITION_REQUEST_PAYMENT_AFTER_ENQUIRY]: STATE_PREAUTHORIZED,
+       },
+     },
+-
+-    [STATE_PENDING_PAYMENT]: {
+-      on: {
+-        [TRANSITION_EXPIRE_PAYMENT]: STATE_PAYMENT_EXPIRED,
+-        [TRANSITION_CONFIRM_PAYMENT]: STATE_PREAUTHORIZED,
+-      },
+-    },
+-
+-    [STATE_PAYMENT_EXPIRED]: {},
+     [STATE_PREAUTHORIZED]: {
+       on: {
+         [TRANSITION_DECLINE]: STATE_DECLINED,
+
+```
+
+###2. Remove Stripe checks from EditListingWizard
+
+In
+[`EditListingWizard.js`](https://github.com/sharetribe/ftw-daily/blob/master/src/components/EditListingWizard/EditListingWizard.js)
+we are checking if the provider has a Stripe account with all the required
+information before we allow them to publish listings. This check is done in the
+`handlePublishListing` function and needs to be removed.
+
+```js
+  handlePublishListing(id) {
+    const { onPublishListingDraft} = this.props;
+    onPublishListingDraft(id);
+  }
+```
+
+After changing the `handlePublishListing` you can remove unused code. You also
+might want to clean up `EditListingPage` and remove the Stripe related props we
+pass to `EditListingWizard`.
+
+###3. Edit ModalMissingInformation
+
+By default,
+[`ModalMissingInformation`](https://github.com/sharetribe/ftw-daily/blob/master/src/components/ModalMissingInformation/ModalMissingInformation.js)
+will remind users to create a Stripe account. The same modal is used for
+reminding about email verification so we should just remove the Stripe related
+code from the component.
+
+###4. Edit CheckoutPage
+
+Removing Stripe from `CheckoutPage` is probably the most complicated task in
+this list because the Strong Customer Authentication (SCA) and ability to save
+payment method have added a lot of logic to the current page. We can use the
+[edited version of CheckoutPage from FTW 2.17.1](/tutorial-assets/SimpleCheckoutPage.js)
```

Comment:
Don't hide a new resource as a link inside paragraph. It's easier to spot these if they are more prominent.

----

We can use the edited version of CheckoutPage from FTW 2.17.1 as a simpler starting point:

- [SimpleCheckoutPage.js](/tutorial-assets/SimpleCheckoutPage.js)